assets/package.json

```diff
@@ -34,10 +35,10 @@
     "zod": "^3.22.4"
   },
   "resolutions": {
-    "@fishjam-dev/ts-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/ts-client&head=main"
+    "@fishjam-dev/ts-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/ts-client&head=fix-install-from-branch"
   },
   "overrides": {
-    "@fishjam-dev/ts-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/ts-client&head=main"
+    "@fishjam-dev/ts-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/ts-client&head=fix-install-from-branch"
```

Comment:
**question (blocking):** What exactly does it do and why do we apply a non-main branch?

Reply:
Because the version on the main doesn't work correctly, more information can provide @kamil-stasiak.

Reply:
There was a problem with the main branch, so we tested a fix on this branch. Now that it's merged, you can use the main branch again